src/components/postgresConnect.ts

```diff
@@ -238,6 +239,24 @@ class PostgresDB {
       return false
     }
   }
+
+  public setApplicationOutcome = async (
+    appId: number,
+    outcome: ApplicationOutcome
+  ): Promise<boolean> => {
+    // Assumes that `is_active` should become False when outcome is set to "Approved" or "Rejected"
```

Comment:
Do you think this could be a function in SQL? So when we change the outcome to `Approve` or `Rejected` the application is automatically changed to have `is_active` as false. I was thinking that leaving this to be an action might be not the best option. In other words: something to be remembered all the time that should be default in the system ideally is a database trigger.

Reply:
Good point, I'll add the postgres trigger now and remove it from this file.

Reply:
Done :)